### PR TITLE
Add a test for trying returndatacopy 0 0 on an empty return data buffer

### PR DESCRIPTION
**Pull request by pirapira**

### CODE REVIEW

File: src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_initial_zero_readFiller.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                "balance" : "0x6400000000",
+                "code" : "",
+                "nonce" : "0x00",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" : {
+            "data" : [
+                ""
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_initial_zero_readFiller.json]: maybe also try it with non empty data? 
"",
"0xffaabb1123"

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_initial_zero_readFiller.json]: I will add that.

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatasize_initial_zero_readFiller.json]: Done.
